weave

```diff
@@ -676,9 +676,14 @@ case "$COMMAND" in
         http_call $DNS_CONTAINER_NAME $DNS_HTTP_PORT GET /status 2>/dev/null || true
         ;;
     ps)
-        [ $# -eq 0 ] && CONTAINERS="$(docker ps -q)" || CONTAINERS="$@"
+        [ $# -eq 0 ] && CONTAINERS="$(echo weave:expose; docker ps -q)" || CONTAINERS="$@"
         for CONTAINER_ID in $CONTAINERS ; do
-            if CONTAINER_ADDRS=$(with_container_netns $CONTAINER_ID container_weave_addrs 2>&1) ; then
+            if [ "$CONTAINER_ID" == "weave:expose" ]; then
```

Comment:
I am guessing that the weavexec container has a more forgiving shell than mine (terrible idea, imo)...

````
$ weave ps
weave:expose 7a:02:94:e9:b9:c0
e0d25927e1b5 92:da:db:36:82:eb
$ sudo weave --local ps
/usr/local/bin/weave: 681: [: weave:expose: unexpected operator
/usr/local/bin/weave: 681: [: e0d25927e1b5: unexpected operator
e0d25927e1b5 92:da:db:36:82:eb
````
Hint: it's `=`, not `==`.